lib/format/kdbx.js

```diff
@@ -435,6 +466,16 @@ Kdbx.prototype._getMasterKey = function(callback) {
     });
 };
 
+Kdbx.prototype._readXml = function(data, callback) {
```

Comment:
change to sync call, as long as there's no async code inside. this will also allow to simplify loading from string